README.md

```diff
@@ -20,3 +20,24 @@ This is the repository for the BOUN SWE Group 4.
 ---
 
 You can check our [wiki](https://github.com/bounswe/bounswe2024group4/wiki) for further information.
+
+
+To run the project, put the following as .env in the root folder:
+
+```
+DB_HOST=db
+DB_NAME=fitness_database
+DB_USER=root
+DB_PASSWORD=admin_group4_dbfitness123
+DB_PORT=3306
+DB_ROOT_PASSWORD=admin_group4_dbfitness123
+EXERCISES_API_KEY=<Your API Ninjas Exercises API Key>
+```
+
+You can get your personal `Exercises API` key from `Ninjas API`s website.
```

Comment:
maybe we can add the link to get the api key

Reply:
Added it to the PR now